web/themes/custom/novel/templates/layout/html.html.twig

```diff
@@ -29,12 +29,20 @@
   ]
 %}
 <!DOCTYPE html>
-<html{{ html_attributes }}>
-  <head>
-    <head-placeholder token="{{ placeholder_token }}">
-    <title>{{ head_title|safe_join(' | ') }}</title>
-    <css-placeholder token="{{ placeholder_token }}">
-    <js-placeholder token="{{ placeholder_token }}">
+<html{{html_attributes}}>
+	<head>
+		<head-placeholder token="{{ placeholder_token }}">
+			<title>{{ head_title|safe_join(' | ') }}</title>
+			<link rel="preload" fetchpriority="low" href="/themes/custom/novel/assets/dpl-design-system/fonts/gochi-hand-v19-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" fetchpriority="low" href="/themes/custom/novel/assets/dpl-design-system/fonts/lora-v26-latin-italic.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/lora-v26-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-500.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-700.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preload" href="/themes/custom/novel/assets/dpl-design-system/fonts/noto-sans-jp-v42-latin-regular.woff2" as="font" type="font/woff2" crossorigin="anonymous"/>
+			<link rel="preconnect" href="http://cover.dandigbib.org"/>
```

Comment:
I really think that these `preconnect` and `preload` urls should be configurable in drupal backend as they may change or be updated (or more being added later on).

Reply:
I agree. And I think we should do it. But considering the amount of time already put into this I think we should write a TODO here and get it done in the future.

Reply:
Then make it a big TODO :-D

Reply:
Like this?:
```
    _____                   _______                   _____                   _______                 
     /\    \                 /::\    \                 /\    \                 /::\    \                
    /::\    \               /::::\    \               /::\    \               /::::\    \               
    \:::\    \             /::::::\    \             /::::\    \             /::::::\    \              
     \:::\    \           /::::::::\    \           /::::::\    \           /::::::::\    \             
      \:::\    \         /:::/~~\:::\    \         /:::/\:::\    \         /:::/~~\:::\    \            
       \:::\    \       /:::/    \:::\    \       /:::/  \:::\    \       /:::/    \:::\    \           
       /::::\    \     /:::/    / \:::\    \     /:::/    \:::\    \     /:::/    / \:::\    \          
      /::::::\    \   /:::/____/   \:::\____\   /:::/    / \:::\    \   /:::/____/   \:::\____\         
     /:::/\:::\    \ |:::|    |     |:::|    | /:::/    /   \:::\ ___\ |:::|    |     |:::|    |        
    /:::/  \:::\____\|:::|____|     |:::|    |/:::/____/     \:::|    ||:::|____|     |:::|    |        
   /:::/    \::/    / \:::\    \   /:::/    / \:::\    \     /:::|____| \:::\    \   /:::/    /         
  /:::/    / \/____/   \:::\    \ /:::/    /   \:::\    \   /:::/    /   \:::\    \ /:::/    /          
 /:::/    /             \:::\    /:::/    /     \:::\    \ /:::/    /     \:::\    /:::/    /           
/:::/    /               \:::\__/:::/    /       \:::\    /:::/    /       \:::\__/:::/    /            
\::/    /                 \::::::::/    /         \:::\  /:::/    /         \::::::::/    /             
 \/____/                   \::::::/    /           \:::\/:::/    /           \::::::/    /              
                            \::::/    /             \::::::/    /             \::::/    /               
                             \::/____/               \::::/    /               \::/____/                
                              ~~                      \::/____/                 ~~                      
                                                       ~~

Reply:
That's beautiful.... let's go with that...